kotsadm/web/src/components/snapshots/ConfigureSnapshots.jsx

```diff
@@ -1,56 +1,65 @@
 import React from "react";
+import Modal from "react-modal";
 import SnapshotInstallationBox from "./SnapshotInstallationBox";
 
 export default function ConfigureSnapshots(props) {
-  const { snapshotSettings, hideCheckVeleroButton, fetchSnapshotSettings, renderNotVeleroMessage, isLicenseUpload, history } = props;
+  const { configureSnapshotsModal, toggleConfigureModal, snapshotSettings, hideCheckVeleroButton, fetchSnapshotSettings, renderNotVeleroMessage } = props;
 
   return (
-    <div className="flex1 flex-column AppSnapshotsEmptyState--wrapper">
-      {isLicenseUpload && <div className="u-fontSize--normal u-fontWeight--medium u-color--royalBlue u-cursor--pointer" onClick={() => history.goBack()}>
-        <span className="icon clickable backArrow-icon u-marginRight--10" style={{ verticalAlign: "0" }} />
-            Back to license upload
-        </div>}
-      <p className={`u-fontSize--largest u-fontWeight--bold u-color--tundora u-marginBottom--10 ${isLicenseUpload && "u-marginTop--12"}`}>Configure application snapshots</p>
-      <p className="u-fontSize--normal u-fontWeight--medium u-color--dustyGray u-lineHeight--normal">
-        In order to configure and use Snapshots (backup and restore), please install <a href="https://kots.io/kotsadm/snapshots/" target="_blank" rel="noopener noreferrer" className="replicated-link">Velero</a> to the cluster. Once Velero is installed, click the button below and the Admin Console will verify the installation and begin configuring Snapshots.
+    <Modal
+      isOpen={configureSnapshotsModal}
+      shouldReturnFocusAfterClose={false}
+      onRequestClose={() => { toggleConfigureModal(); }}
+      ariaHideApp={false}
+      contentLabel="Modal"
+      className="Modal ConfigureSnapshots"
+    >
+      <div className="Modal-body">
+        <p className="u-fontSize--largest u-fontWeight--bold u-color--tundora u-marginBottom--10 u-marginTop--12">Add a new destination</p>
+        <p className="u-fontSize--normal u-fontWeight--medium u-color--dustyGray u-lineHeight--normal">
+          In order to configure and use Snapshots (backup and restore), please install <a href="https://kots.io/kotsadm/snapshots/" target="_blank" rel="noopener noreferrer" className="replicated-link">Velero</a> to the cluster. Once Velero is installed, click the button below and the Admin Console will verify the installation and begin configuring Snapshots.
```

Comment:
```suggestion
          In order to configure and use Snapshots (backup and restore), please install <a href="https://kots.io/kotsadm/snapshots/" target="_blank" rel="noopener noreferrer" className="replicated-link">Velero</a> in the cluster. Once Velero is installed, click the button below and the Admin Console will verify the installation and begin configuring Snapshots.
```

Reply:
What does Admin Console actually do to "configure" the snapshots after verifying? I'm guessing it's nothing, so we should strike that.

```suggestion
          In order to configure and use Snapshots (backup and restore), please install <a href="https://kots.io/kotsadm/snapshots/" target="_blank" rel="noopener noreferrer" className="replicated-link">Velero</a> in the cluster and configure it for your cloud storage provider. Once Velero is configured, click the button below and the Admin Console will verify the installation.
```